.gitmodules

```diff
@@ -1,21 +1,15 @@
-[submodule "amigos/ext/PyCampbellCR1000"]
-	path = amigos/ext/PyCampbellCR1000
-	url = https://github.com/LionelDarras/PyCampbellCR1000
-[submodule "amigos/ext/pynmea2"]
-	path = amigos/ext/pynmea2
-	url = git@github.com:Knio/pynmea2.git
-[submodule "amigos/ext/requests"]
-	path = amigos/ext/requests
-	url = https://github.com/kennethreitz/requests.git
-[submodule "amigos/ext/amigos/ext/urllib3"]
-	path = amigos/ext/urllib3
-	url = https://github.com/urllib3/urllib3.git
-[submodule "amigos/ext/idna"]
-	path = amigos/ext/idna
-	url = https://github.com/kjd/idna.git
-[submodule "amigos/ext/chardet"]
-	path = amigos/ext/chardet
-	url = https://github.com/chardet/chardet.git
-[submodule "amigos/ext/PyLink"]
-	path = amigos/ext/PyLink
-	url = https://github.com/LionelDarras/PyLink.git
+[submodule "amigos/amigos/pylink"]
```

Comment:
I would rather we kept these external libraries in their own directory (e.g. 'amigos/ext' like before) just to separate *our* code from external libraries.  These shouldn't change too much, and if they do it will be on their own repositories -- if they are under 'amigos' then they clutter up things when trying to work on 'amigos'.

Reply:
There is also the 'onvif' library committed, shouldn't it be a submodule?